pkg/artifactory/datasource_artifactory_file.go

```diff
@@ -0,0 +1,202 @@
+package artifactory
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"github.com/atlassian/go-artifactory/v2/artifactory"
+	"github.com/atlassian/go-artifactory/v2/artifactory/v1"
+	"github.com/hashicorp/terraform/helper/schema"
+	"io"
+	"os"
+)
+
+func datasourceArtifactoryFile() *schema.Resource {
+	return &schema.Resource{
+		Create: nil,
```

Comment:
You don't need to specify `Create` here as it will default to `nil`

Reply:
Resolved with 0ca7f19